### DISCUSSION TITLE
Can I use SelfControl to lock me out of the computer once the timer runs out?

### DISCUSSION THREAD

**Discussion by stereokai** [SelfControlApp/selfcontrol#779] [Q&A]
I would like to force myself to take breaks every once in a while, and thought a good way to do it would be to lock me out of the computer for a few moments every half hour or an hour etc.

Is it possible to use SelfControl for this? (one easy integration would be to have it run a command as soon as the timer runs out)

Cheers for help!

**Comment by slambert** [comment]
Yes - check this out: 

https://github.com/andreasgrill/auto-selfcontrol

**Comment by stereokai** [comment]
@slambert Thanks a lot! But I guess my question was too cryptic because I'm not sure that solves my issue.
I want to use SelfControl manually (with the build in timer), but instead of blocking the internet, I would like it to lock me out of my computer for 5 or 10 minutes as soon as the timer runs out.

Do you think something like that is possible?
